apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -384,6 +385,12 @@ public void accept(ErrorReport data) {
         listView.setOnItemClickListener(TestFeatures.getOnItemClickListener());
     }
 
+    /* Get the default app secret from the app secret array. */
+    protected static String GetDefaultAppSecret(Resources resources) {
```

Comment:
package-private is better (no keyword), protected is needed when subclassing.

Also this is a C# method name, while this is Java, it needs to be camelCase.